source/sphinx_extensions/command_block/extension.py

```diff
@@ -40,17 +63,34 @@ class CommandBlockDirective(docutils.parsers.rst.Directive):
     has_content = True
```

Comment:
Interesting, I thought this declaration would make `.. download::` fail, since it doesn't have any content. Any ideas? Maybe it's worth removing the line if it isn't doing anything.